src/components/ResultList/SimpleResultList.tsx

```diff
@@ -38,6 +42,33 @@ export const SimpleResultList = (props: ISimpleResultListProps): ReactElement =>
 
   const { highlights, showHighlights, isFetchingHighlights } = useHighlights();
 
+  const { settings } = useSettings();
+  const { defaultExportFormat, customFormats } = settings;
+
+  const bibcodes = docs.map((d) => d.bibcode).sort();
+
+  const { data: citationData } = useGetExportCitation(
+    {
+      // format: values(exportFormats).find((f) => f.label === defaultExportFormat).id,
+      format: ExportApiFormatKey.agu,
+      customFormat: defaultExportFormat === exportFormats.custom.label ? customFormats[0].code : undefined,
+      bibcode: bibcodes,
+      sort: ['bibcode asc'],
+    },
+    { enabled: !!settings?.defaultExportFormat },
+  );
+
+  // a map from bibcode to citation
+  const defaultCitations = useMemo(() => {
+    const citationSet = new Map<string, string>();
+    if (!!citationData) {
```

Comment:
We should do better error checking here.  And possibly wrap with try/catch just in case

Reply:
@thostetler I added try catch, and reset the map to empty if error.